gateway/perf/performance-test-scripts/jmeter/run-scenario.sh

```
@@ -0,0 +1,361 @@
+    local response_size="${rsize:-10240}"
+    # EKS / in-cluster mock: gateway reaches ClusterIP; no SSH to a backend EC2.
+    if [[ "${BACKEND_IN_CLUSTER:-}" == "1" || -z "${BACKEND_SSH:-}" ]]; then
+        echo "Skipping remote backend start (BACKEND_IN_CLUSTER=${BACKEND_IN_CLUSTER:-0}, BACKEND_SSH=${BACKEND_SSH:-<empty>})"
+        return 0
+    fi
+    local ssh_opts=(-o StrictHostKeyChecking=no -o ConnectTimeout=10)
+    [[ -f "${BACKEND_SSH_KEY}" ]] && ssh_opts+=(-i "${BACKEND_SSH_KEY}")
+    if [[ "${BACKEND_IMPL:-jar}" == "compose" || "${BACKEND_IMPL:-jar}" == "docker" ]]; then
+        echo "Ensuring container backend is up (impl=${BACKEND_IMPL:-jar}, port=${BACKEND_PORT}) — skip jar reconfigure"
+        ssh "${ssh_opts[@]}" "${BACKEND_SSH}" \
+            "cd ${PERF_ROOT}/ai-gateway-manual/backend && \
```

Shellcheck (0.11.0)

[info] Note that, unescaped, this expands on the client side.

(SC2029)